src/process_utility.c

```diff
@@ -365,8 +365,22 @@ process_copy(ProcessUtilityArgs *args)
 	Cache *hcache;
 	Oid relid;
 
+	/* We only copy for COPY FROM. Since hypertables do not contain any data,
+	 * we skip the copy for COPY TO, but print an informative message. */
 	if (!stmt->is_from || NULL == stmt->relation)
+	{
+		if (stmt->relation)
+			ereport(NOTICE,
```

Comment:
I actually think this can be made an `ERROR`, which would be the right semantics for hypertables, or make this work underneath the hood if possible. But I guess the issue was that this will break `pg_dump`?

Reply:
Exactly, it will break all tools, which use `COPY TO`. At least, we are aware about `pg_dump`.